src/test/javascript/spec/component/file-upload-assessment/file-upload-assessment.component.spec.ts

```diff
@@ -249,10 +255,11 @@ describe('FileUploadAssessmentComponent', () => {
         const submission = createSubmission(exercise);
         const feedback = new Feedback();
```

Comment:
A function that creates and returns these feedbacks would be much more compact.

Reply:
I think this is just fine :)